addons/sale_timesheet/models/project.py

```diff
@@ -320,6 +363,33 @@ def get_profitability_common(self):
             'timesheet_percentage_billable': timesheet_billable and timesheet_billable / (timesheet_billable + timesheet_non_billable) * 100 or 0.0,
         }
 
+    def _get_stat_buttons(self):
+        buttons = super(Project, self)._get_stat_buttons()
+        buttons.append({
+            'icon': 'clock-o',
+            'text': 'Billable Time',
+            'number': '%s %%' % (self.billable_time),
+            'action_type': 'action',
+            'action': 'hr_timesheet.act_hr_timesheet_line_by_project',
+            'additional_context': json.dumps({
+                'active_id': self.id,
+                'search_default_project_id': self.id,
+                'search_default_groupby_timesheet_invoice_type': True
+            }),
+            'show': self.show_billable_time_button,
+            'order': 9
+        })
+        buttons.append({
+            'icon': 'pencil-square-o',
+            'text': 'Invoices',
+            'number': self.invoices_count,
+            'action_type': 'object',
+            'action': 'action_open_project_invoices',
+            'show': self.show_invoices_count_button,
+            'order': 11
+        })
+        return sorted(buttons, key=lambda k: k['order'])
```

Comment:
Instead of reordering at each super call, you could order them when calling the method nope ?